### DISCUSSION TITLE
16.4.0

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#872] [General]
## Releases Notes for 16.4.0
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v16.4.0/draw.io-16.4.0-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v16.4.0/draw.io-16.4.0-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v16.4.0/draw.io-universal-16.4.0.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v16.4.0/drawio-amd64-16.4.0.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v16.4.0/drawio-x86_64-16.4.0.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v16.4.0/drawio-x86_64-16.4.0.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Fixes #862 , #547 , #864 
- Updates to [draw.io core 16.4.0](https://github.com/jgraph/drawio/blob/v16.4.0/ChangeLog). All changes from 16.1.0 to 16.4.0, inclusive, are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v16.4.0'>16.4.0</a>.</em>

**Comment by Callez** [comment]
![image](https://user-images.githubusercontent.com/6614431/150511176-a98a9eb2-8495-4e6c-803f-7f72b3a9375b.png)
Hello, 
when installing Windows Installer version, the installation fails on last phase to LNK file error . I´m installing this within non-admin account via "run as Admin" option.



Can this be fixed ?

**Comment by educoutinho** [comment]
In my computer, this new version is extremely slower than the previous version.

**Reply by putnik2000** [reply]
I agree, in my case the same:  on windows 10 (10.0.19043) the latest version is extremely slow both during program load and on any action with drawing. Rolled back to 16.1.2 - works perfectly.

**Comment by davidjgraph** [comment]
Please file bugs for bugs https://github.com/jgraph/drawio-desktop/issues/new?assignees=&labels=&template=bug_report.md

**Reply by Callez** [reply]
Now filed to bug report.

Btw, none of the installation files doesnt bring the solution. I´ve triad also by logging in with admin account and uninstalled & reinstalled few times. Now I start manual cleaning process to remove all remains of Drawio 

Steps done:
Msiexec/uninstall draw.io-16.4.0.msi

Manually removed:
- All Draw.io related files on computer
- All Draw.io references from registry
- Check Updates

msiexec /i draw.io-16.4.0.msi ALLUSERS=1 /qn

-Same results	

msiexec /uninstall draw.io-16.4.0.msi ALLUSERS=1 /qn

Elevated my work account to Admin
Tried draw.io-16.4.0-windows-installer.exe, but same LNK error

Trid to manually launch C:\Program Files\draw.io\drawio.exe - Reports that no enoug rights to run it

Uninstalling again..manually, since cannot uninstall from add remove .. (see attachment)

Reboot

At this point I´ll surrender and use the below version, since it seems to be the only one which work after the unsuccessful installation attems.

draw.io-16.0.2-windows-no-installer.exe

**Comment by danieltw** [comment]
draw.io-16.4.0-windows-no-installer.exe, why does it always open two windows during execution?
![error](https://user-images.githubusercontent.com/2370912/150710692-b1c8c506-39e7-4cd9-9d28-f958a5bae0e0.jpg)
